src/plugins/visualizations/public/vis_types/types_service.ts

```diff
@@ -71,17 +69,17 @@ export class TypesService {
     return {
       /**
        * registers a visualization type
-       * @param {VisType} config - visualization type definition
+       * @param config - visualization type definition
        */
-      createBaseVisualization: (config: any) => {
+      createBaseVisualization: (config: BaseVisTypeOptions): void => {
```

Comment:
:information_source: This is the most important part of this PR, since it finally puts the proper typings for the visualization registry into the public API we expose.